compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangExpression.java

```diff
@@ -51,4 +52,6 @@
     public BType expectedType;
 
     public Map<BVarSymbol, NarrowedTypes> narrowedTypeInfo;
+
+    public BSymbol symbol;
```

Comment:
Wouldn't adding a symbol to `BLangExpression` be a bit misleading considering a symbol would only be associated in a few specific exprs such as var ref, function call etc? 🤔

Reply:
Yeah AFAICT there don't seem to be places where we used to access the symbol via the previous BLangAccessibleExpression. If we can add it to only the relevant expressions, shall we do so?

Reply:
fixed: 89e2edd